app/src/pages/domainsPage/components/homeDomainCard.tsx

```diff
@@ -30,6 +17,7 @@ export const HomeDomainCard = (props: IHomeDomainCardProps): React.ReactElement
         <Stack direction={Direction.Horizontal} shouldAddGutters={true} childAlignment={Alignment.Center}>
           <Text variant='header3'>{props.homeDomain.domain.url}</Text>
           <Text variant='note'>{props.homeDomain.domain.isVerified ? 'verified' : 'unverified'}</Text>
+          {/* <Pill variant={props.homeDomain.domain.isVerified ? 'success-small' : 'default-small'} text={props.homeDomain.domain.isVerified ? 'verified' : 'unverified'} /> */}
```

Comment:
commented out?

Reply:
the pill is huge at the moment, looks like this 🤦‍♂️
![Screen Shot 2020-12-04 at 10 45 33](https://user-images.githubusercontent.com/3192370/101154629-dfb40300-361d-11eb-9fd5-c293b47dd3d8.png)

Reply:
oh actually i can update the styles here, doesnt need to be in main library, will do now

Reply:
updated, silly me 🤦‍♂️